impeller/entity/geometry.cc

```diff
@@ -16,6 +16,46 @@
 
 namespace impeller {
 
+/// Given a convex polyline, create a triangle fan structure.
+std::pair<std::vector<Point>, std::vector<uint16_t>> TessellateConvex(
+    Path::Polyline polyline) {
+  std::vector<Point> output;
+  output.reserve(polyline.points.size() * 3);
+
+  for (auto j = 0u; j < polyline.contours.size(); j++) {
```

Comment:
I think this needs an assertion that there's only a single contour. Otherwise, please add some tests that show it works with multiple contours.